### PR TITLE
fix: correct spelling errors and improve variable naming for clarity

### PR DESCRIPTION
**Pull request by Coooder-Crypto**
## Summary
This pull request addresses several spelling mistakes and improves variable naming conventions across the codebase to enhance readability and maintainability.

## Changes
- Corrected spelling errors.
- Renamed variables to follow consistent naming conventions.

## Testing
- Verified that all unit tests pass successfully.
- Manually tested key functionalities to ensure no regressions.

## Additional Notes
- No functional changes were made; this PR is purely for code quality improvement.

### CODE REVIEW

File: src/app/creator/build/[type]/ManageModal.js [hunk 1]
```diff
@@ -52,12 +52,12 @@ export function ManageModal({
   closeModal,
   type,
   bounty,
-  succesCallback,
+  successCallback,
```
Comment (ourai) [inline src/app/creator/build/[type]/ManageModal.js:55]: The places used this prop should be updated.

File: package.json [hunk 2]
```diff
@@ -116,5 +116,6 @@
     "postcss": "^8.4.18",
     "prettier": "^2.5.1",
     "tailwindcss": "^3.2.4"
-  }
+  },
+  "packageManager": "pnpm@9.13.2+sha512.88c9c3864450350e65a33587ab801acf946d7c814ed1134da4a924f6df5a2120fd36b46aab68f7cd1d413149112d53c7db3a4136624cfd00ff1846a0c6cef48a"
```
Comment (ourai) [inline package.json:120]: This field doesn't need to be specified for now.

**Review by ourai — CHANGES_REQUESTED** [review]
This PR is very helpful, thank you! There're two places need to change.

**Comment by Coooder-Crypto** [comment]
Thanks for the feedback! I have revised the code~

**Pull request merged**